cms/core/models/snippets.py

```diff
@@ -26,11 +27,24 @@ class ContactDetails(index.Indexed, models.Model):
         index.SearchField("name"),
         index.AutocompleteField("name"),
         index.SearchField("email"),
+        index.AutocompleteField("email"),
         index.SearchField("phone"),
     ]
 
     class Meta:
-        verbose_name_plural = "contact details"
+        verbose_name = _("contact details")
+        verbose_name_plural = _("contact details")
+        constraints: ClassVar[list[models.BaseConstraint]] = [
+            models.UniqueConstraint(
+                name="%(app_label)s_%(class)s_name_unique",
+                fields=["name", "email"],
+            ),
+        ]
```

Comment:
I'm not sure what the answer should be, but these are currently case-sensitive. It feels more appropriate if they were case-insensitive checks; otherwise, you can have tw that look almost the same as long as they differ in case.

Reply:
Nice one, small issue, the error message is not useful to the end user. It currently displays:

<img width="438" alt="image" src="https://github.com/user-attachments/assets/2a2895e6-6e82-43d5-84f6-8f2e147ff8b1">